docs/root/dev/writing-intel-modules.md

```diff
@@ -103,52 +232,12 @@ All of these queries use indexes for faster lookup.
 #### Create an index for new nodes
 
 Be sure to [update the indexes.cypher file](https://github.com/lyft/cartography/blob/8d60311a10156cd8aa16de7e1fe3e109cc3eca0f/cartography/data/indexes.cypher)
-with your new node type. Indexing on ID is required, and indexing on anything else that will be frequently queried is
+with your new node type. Indexing on "`id`" is required, and indexing on anything else that will be frequently queried is
 encouraged.
 
 #### lastupdated and firstseen
 
-Set the `lastupdated` and `firstseen` fields on both nodes and relationships. Suppose we are creating
-the following chain:
-
-```cypher
-MERGE (n:NodeType)-[r:RELATIONSHIP]->(n2:NodeType2)
-```
-
-- To handle nodes in this case,
-
-    - Every `MERGE` query that creates a new node should look like this
-
-        ```cypher
-        ON CREATE SET n.firstseen = $UpdateTag
-        SET
-        n.lastupdated = $UpdateTag,
-        node.field1 = $value1,
-        node.field2 = $value2,
-        ...
-        node.fieldN = $valueN
-        ```
-
-- To handle relationships in this case,
-
-    - Every `MERGE` query that creates a new relationship should look like this
-
-        ```cypher
-        ON CREATE SET r.firstseen = $UpdateTag
-        SET
-        r.lastupdated = $UpdateTag
-        ```
-
-#### Connecting different node types with the `_attach` pattern
```

Comment:
I feel like their may be cases where we want to attach after loading. But in the cases I can think of, it's still best to transform the data and use a NodeSchema that properly defines those extra relationships.

Reply:
Good point, and yes that's the approach I would encourage.

It's also possible to first generate a query using `selected_relationships` and then load to that with additional data, but I don't have an example ready for that and that sounds complicated and I'm not ready to encourage that pattern in the doc yet.